12_upgrade/20_upgrade_centos_rhel.txt

```diff
@@ -32,7 +32,7 @@ For Rudder Server (RHEL 6 only), upgrade the +rudder-*+ and +ncf+ packages:
 
 ----
 
-yum update "rudder-*" "ncf"
+yum update "rudder-*" "ncf-*"
```

Comment:
No, if you do this you will not update ncf anymore (you match ncf >-< *)

The correct command would be:
```
yum update "rudder-*" "ncf*"
```